targets/avr.json

```diff
@@ -9,7 +9,8 @@
 	"scheduler": "none",
 	"default-stack-size": 256,
 	"cflags": [
-		"--target=avr-unknown-unknown"
+		"--target=avr-unknown-unknown",
+		"-Werror"
```

Comment:
I've added this line because I've seen warnings recently regarding unused parameters in AVR builds that weren't caught before. This commit fixes these warnings (by adding `-Qunused-arguments`) and with this flag also ensures that the compilation will fail if this happens again.

Reply:
:+1: